polkadot/xcm/src/lib.rs

```diff
@@ -398,25 +599,38 @@ impl WrapVersion for AlwaysV2 {
 pub struct AlwaysV3;
 impl WrapVersion for AlwaysV3 {
 	fn wrap_version<Call>(
-		_: &latest::MultiLocation,
+		_: &latest::Location,
 		xcm: impl Into<VersionedXcm<Call>>,
 	) -> Result<VersionedXcm<Call>, ()> {
 		Ok(VersionedXcm::<Call>::V3(xcm.into().try_into()?))
 	}
 }
 
+/// `WrapVersion` implementation which attempts to always convert the XCM to version 3 before
+/// wrapping it.
+pub struct AlwaysV4;
+impl WrapVersion for AlwaysV4 {
+	fn wrap_version<Call>(
+		_: &latest::Location,
+		xcm: impl Into<VersionedXcm<Call>>,
+	) -> Result<VersionedXcm<Call>, ()> {
+		Ok(VersionedXcm::<Call>::V4(xcm.into().try_into()?))
+	}
+}
+
 /// `WrapVersion` implementation which attempts to always convert the XCM to the latest version
 /// before wrapping it.
-pub type AlwaysLatest = AlwaysV3;
+pub type AlwaysLatest = AlwaysV4;
 
 /// `WrapVersion` implementation which attempts to always convert the XCM to the most recent Long-
 /// Term-Support version before wrapping it.
-pub type AlwaysLts = AlwaysV3;
+pub type AlwaysLts = AlwaysV4;
```

Comment:
Long-term support doesn't necessarily need to update with the latest. Did you consult with someone else about this?

Reply:
No, I can leave it as v3, but v4 is mostly v3 with syntax we'll want to support long-term